build.gradle

```diff
@@ -3,15 +3,19 @@
 buildscript {
     ext.kotlin_version = '1.3.50'
     ext.objectboxVersion = '2.5.1'
+    ext.ktlint_version = '9.2.1'
```

Comment:
Can this be put in the app/build.gradle or does it have to be here?

Reply:
I think it has to be there, since the plugin is installed project-wide